Project2.md

```diff
@@ -58,13 +54,68 @@ but I think it's a reasonable short-cut for a class project. The risk
 with this is that it could expose security vulnerabilities so you
 wouldn't want to really put it into production.
 
+Docker Compose
+--------------
+
+One other thing to introduce before we start coding is Docker Compose.
+Compose is a tool for defining the topology and running multi-container Docker applications. 
+With Compose, you use a Compose file to configure your application’s services.
+Recall how we start web, mysql containers one by one in the previous project. This
+approach works fine when we have only two containers and the configuration is simple, but
+is certainly not scalable. Using Docker Compose, we will be able to stand up our containers
+using a single command. Here is a tutorial about how Docker Compose works:
+
+First, install Docker Compose by doing
+
+	$ curl -L https://github.com/docker/compose/releases/download/1.8.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+	$ chmod +x /usr/local/bin/docker-compose
+
+
+When you've got Compose installed, create a new directory with
+
+	mkdir Compose
+	cd Compose
+
+Then go ahead and create a new file, docker-compose.yml, with your favorite text editor. Docker Compose uses these files, called YAML files, to automatically set up or start a group of containers. This is handy, much easier than managing each container individually. You can specify a lot of different options here, but for now we're merely going to create our entity layer and then link it to our mysql database. You need to make sure the mysql container we created from last time is started.
+
+You can tell compose to create a new container by giving it a name and an image to use.
+
+	models:
+	    image: tp33/django:1.3
```

Comment:
I don't believe that version on the container is correct. Note that in project 1 there are no versions.